lang/LangPrimSource/SC_CoreMIDI.cpp

```diff
@@ -650,7 +655,19 @@ int prInitMIDI(struct VMGlobals* g, int numArgsPushed) {
     if (err)
         return errWrongType;
 
-    return initMIDI(numIn, numOut);
+    err = initMIDI(numIn, numOut);
+
+    // capture initialization times to compute offsets
+    gLogicalInitTime = logicalTimeToNanos(g);
+    // do this last to minimize offset from logical
+
+#if SC_IPHONE
```

Comment:
If we don't have anything to solve for the SC_IPHONE branch right now, I'd prefer just getting rid of this for now, or implementing what is needed.